### ISSUE TITLE
convert only_metadata.ltxml to Beilstein XML

### ISSUE THREAD

**Issue by jbs1** [jbs1/tract2#131]
_migrated from Trac, where originally posted by **kohlhase** on 2-May-2013 3:53pm_

Building on #1477, we need to write an XSLT style sheet that transforms `only_metadata.ltxml` into Beilstein XML. 

MiKo will supervise this and make an empty stylesheet and a Makefile entry.

**Comment by jbs1** [comment]
_migrated from Trac, where originally posted by **kohlhase** on 2-May-2013 4:05pm_

I made a rule in the Makefile to make `*.beil` (Beilstein XML files), e.g. `only_metadata.beil` (committed). Lukas, you just need to add templates to `ltx2beil.xsl`. BTW, I made a convenient fallback template that gives you error messages which templates are still unimplemented. 

To learn about the target format look at `resources/beilstein_beispiel.xml` and `resources/Creating_Beilstein_XMLFile.pdf`.
